addons/hr_attendance/models/hr_employee.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+    _description = "Employee"
+
+    state = fields.Selection([('absent', 'Absent'), ('present', 'Present')], compute='_state', string='Attendance')
+    last_sign = fields.Datetime(compute='_last_sign', string='Last Sign')
+    attendance_access = fields.Boolean(compute='_attendance_access', string='Attendance Access')
```

Comment:
There is no other use of attendance_access except checking in javascript that whether user having base.group_hr_attendance or not, we can achieve by calling has_group method and show element only if user having group base.group_hr_attendance, @hmo-odoo what do you think, should we keep this field or move this checking stuff in js ?